drivers/d3d12/rendering_device_driver_d3d12.cpp

```diff
@@ -1815,8 +1824,17 @@ RDD::CommandBufferID RenderingDeviceDriverD3D12::command_buffer_create(CommandBu
 	// Bookkeep
 
 	CommandBufferInfo *cmd_buf_info = VersatileResource::allocate<CommandBufferInfo>(resources_allocator);
-	cmd_buf_info->cmd_allocator = cmd_allocator;
-	cmd_buf_info->cmd_list = cmd_list;
+	cmd_buf_info->cmd_allocator.Attach(cmd_allocator);
+	cmd_buf_info->cmd_list.Attach(cmd_list);
+
+	RBMap<CommandPoolID, LocalVector<CommandBufferInfo *>>::Element* elem = pools_command_buffers.find(p_cmd_pool);
+	if (elem != nullptr) {
+		elem->value().push_back(cmd_buf_info);
+	} else {
+		LocalVector<CommandBufferInfo *> command_buffer_infos;
+		command_buffer_infos.push_back(cmd_buf_info);
```

Comment:
```suggestion
		LocalVector<CommandBufferInfo *> command_buffer_infos({ cmd_buf_info });
```
Might read better, just a suggestion 🙂